pkgs/development/python-modules/nengo/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, fetchFromGitHub, buildPythonPackage,
+  numpy,
+  scipySupport ? false, scipy,
+  scikitSupport ? false, scikit-learn,
```

Comment:
generally it's preferred to have leading `,`'s, to avoid larger diffs
```suggestion
{ lib, fetchFromGitHub, buildPythonPackage
, numpy
, scipySupport ? false, scipy
, scikitSupport ? false, scikit-learn
```

Reply:
cc @FRidh on handling optional dependencies